zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/repo/NotebookRepoSync.java

```diff
@@ -92,6 +94,10 @@ public NotebookRepoSync(ZeppelinConfiguration conf) {
       LOG.info("No storages could be initialized, using default {} storage", defaultStorage);
       initializeDefaultStorage(conf);
     }
+    //syncOnStart();
```

Comment:
@rajarajan-g thanks for pointing to it. actually initial purpose was to delete it, but it was extracted into sub-function and commented out. addressed by deleting it in 1c11c2ddbd27b222669278f22f87b65cd2292fc9